R/int_pctl.R

```diff
@@ -95,171 +103,146 @@ int_pctl.tune_results <- function(.data, metrics = NULL, eval_time = NULL,
     check_enough_eval_times(eval_time, metrics)
   }
 
-  .data$.predictions <- filter_predictions_by_eval_time(.data$.predictions, eval_time)
+  .data$.predictions <- filter_predictions_by_eval_time(
+    .data$.predictions,
+    eval_time
+  )
 
   y_nm <- outcome_names(.data)
 
-  config_keys <- get_configs(.data, parameters = parameters)
-  p <- length(config_keys)
-  metrics_info <- metrics_info(metrics)
+  config_keys <- get_configs(
+    .data,
+    parameters = parameters,
+    as_list = FALSE
+  )
 
+  metrics_info <- metrics_info(metrics)
   res <-
-    purrr::map2(
-      config_keys, sample.int(10000, p),
-      ~ boostrap_metrics_by_config(.x, .y, .data, metrics, times, allow_par,
-                                   event_level, alpha, metrics_info)
-    ) %>%
-    purrr::list_rbind() %>%
-    dplyr::arrange(.config, .metric)
-  dplyr::as_tibble(res)
-}
-
-get_int_p_operator <- function(allow = TRUE) {
```

Comment:
Git chose to diff these changes very strangely

Reply:
I agree.